### ISSUE TITLE
[NPC][WotLK] Mobs' Chase Range

### ISSUE THREAD

**Issue by warcraftgeekygamer** [dalaranwow/dalaran-wow#4500]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
All mobs of any and all locations have a very short chase range. Currently when you aggro a mob and run for a bit of distance, the mob stops chasing you after about 2m which is not at all "blizz-like".

**Expected behaviour**: Tell us what should happen instead
All mobs of any and all locations should instead chase you for longer than 2m.

**Steps to reproduce the problem**:

1. Locate a mob (location and type of mob does not matter)
2. Aggro a mob
3. Run away from the mob and observe as it will chase you for a very short distance before the mob will reset

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/ Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Proof: Retail WoW. In Leigon, mobs of any level and any location will chase you for longer than 2m before it will reset.

Click the two links ---> http://wowwiki.wikia.com/wiki/Leash and http://wow.gamepedia.com/Aggro_radius

Reference the **Formula** section in the first link.

> Current in-game testing shows the following facts:
> 
> * The aggro radius of a mob having the same level as the player is roughly 20 yards (~ 18 meters)
> 
> * Aggro radius varies with level difference at a rate of roughly 1 yard/level.
> 
> * Minimum aggro radius for a mob seems to be combat range (5 yards).
> 
> * A Hunter's pet seems to ignore the minimum aggro radius for low level mobs, even when controlled through [Eyes of the Beast].
> 
> * Aggro radius is capped at a maximum of 45 yards, that is, when the mob is 25 or more levels higher than the player.
> 
> * Vertical distance seems to be weighted much more heavily than horizontal distance. A mob standing on a ledge 4 yards above your head can fail to aggro even if standing 4 yards away from the same mob on level ground will draw aggro.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
This is already fixed on PTR and needs to be applied to the live realm. It's a server config change only.

```sql
# CREATURE SETTINGS
#
#    ThreatRadius
#        Description: Distance for creatures to evade after being pulled away from the combat
#                     starting point. If ThreatRadius is less than creature aggro radius then aggro
#                     radius will be used.
#        Default:     60

ThreatRadius = 60
```

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
